frontend/src/pages/answer/kgBubble/KgBubble.jsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable indent, no-use-before-define, func-names, no-return-assign */
+import React, {
```

Comment:
I'm controlling the nodes to not go outside the bounds of the box. Though I'm not sure letting the bubbles go just slightly outside the box is what we want. @sarah-seitanakis do you have any input?

Reply:
I agree that if the user is dragging a circle to the edge of the box, it would be more natural to see the bubble push into the edge a little. But, we don't want this behavior when the graph is resting-- only when the user is moving things around.

Reply:
Updated in c53e41393655603c8f82d949f84ca8a652674352. The dragged node can node push into the edges a little while all other nodes are bound inside. Once you let go of the node, it is again constricted to inside.